Add from to use_helpers to add macro like syntax (#2034)
ViewComponent/view_component: docs/guide/helpers.md

```ruby
class UseHelpersComponent < ViewComponent::Base
  use_helpers :icon, :icon?

  erb_template <<-ERB
    <div class="icon">
      <%= icon? ? icon(:user) : icon(:guest) %>
    </div>
  ERB
end
```

> Methods defined in helper modules that aren't available in the component can be included individually using the `from:` keyword:

Reviewer: ```suggestion
Use the `from:` keyword to include individual methods defined in helper modules not available in the component:
```

```ruby
class UserComponent < ViewComponent::Base
  use_helpers :icon, :icon?, from: IconHelper

  def profile_icon
    icon? ? icon(:user) : icon(:guest)
  end
end
```

> The singular version `use_helper` is also available for a single helper:

Reviewer: ```suggestion
The singular version `use_helper` is also available:
```

end
```

Methods defined in helper modules that are not available in the component can be included individually using the `from:` keyword:

```ruby
class UserComponent < ViewComponent::Base

>   use_helpers :icon, from: IconHelper

Reviewer 2: I guess it would make sense to support the singular form `use_helper` as well?

Author: @Spone I have added this recommendation, as it seems like a great fit
